testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadRedirectTestCase.java

```diff
@@ -226,6 +226,7 @@ public void testReloadwithRedirect() throws Exception {
             promptFound = cliProc.pushLineAndWaitForResults("reload", "Accept certificate");
             assertTrue("No certificate prompt " + cliProc.getOutput(), promptFound);
         } finally {
+            cliProc.ctrlCAndWaitForClose();
             cliProc.destroyProcess();
```

Comment:
I noticed that on Solaris10 sparc that cliProc is not completely destroyed using just cliProc.destroyProcess(); - test pass, maven execution successfully ends, but a running cli process remains. Using just CtrlC works fine.